.github/workflows/build-and-publish.yaml

```diff
@@ -67,14 +67,24 @@ jobs:
         with:
           opm: "latest"
 
-      - name: Install yq
+      - name: Install yq for amd64
+        if: ${{ "$(uname -m)" == "amd64" }}
         env:
           VERSION: v4.14.2
           BINARY: yq_linux_amd64
         run: |
           wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O /usr/local/bin/yq
           chmod +x /usr/local/bin/yq
 
+      - name: Install yq for ppc64le
+        if: ${{ "$(uname -m)" == "ppc64le" }}
```

Comment:
We use GitHub-hosted runners, and they are available [only for x86_64](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources).

Also you cannot use shell commands in GitHub expressions `${{ ... }}`, see [GitHub Docs for Expressions](https://docs.github.com/en/actions/learn-github-actions/expressions).

Reply:
How would you like us to proceed regarding workflows if GH runners are only available for x86_64.